source/clear-linux/guides/network/custom-clear-container.rst

```diff
@@ -111,18 +111,19 @@ Build the base Clear Linux container image
       * `os-core-update` provides basic suite for running the |CL|
         for iA Updater
 
-#. Optionally, additional bundles can be included with the base image.   
+   #. Optionally, additional bundles can be included with the base image.
```

Comment:
This line, did not have the indentation issue. Please render the doc to confirm that the numeration renders properly.

Reply:
Misunderstood your initial comment! Resolved!